test/configurationTests.ts

```diff
@@ -251,6 +251,22 @@ describe("Configuration", () => {
             const actualConfig = extendConfigurationFile(baseConfig, extendingConfig);
             assertConfigEquals(actualConfig, expectedConfig);
         });
+
+        it("overrides defaultSeverity of base configs", () => {
+            const config = loadConfigurationFromPath("./test/config/tslint-extends-default-severity.json");
+            assert.equal<RuleSeverity | undefined>(
+                "warning",
+                config.rules.get("default-severity-unspecified")!.ruleSeverity,
+                "did not apply defaultSeverity to base config with no defaultSeverity");
```

Comment:
I was a bit confused because all other tests use the message the other way around. They describe what the test expects, not what caused it to fail. For example: `assert.isNull(err, "process should exit without an error");`

There's only one other test in this file that provides a message for assertions. Please change them too while you are here.